releasenotes/releasenotes_v5.rst

```diff
@@ -7,6 +7,26 @@ Current Firely Server release notes (v5.x)
     For information on how to upgrade, please have a look at our documentation on :ref:`upgrade`. You can download the binaries of the latest version from `this site <https://downloads.fire.ly/firely-server/versions/>`_, or pull the latest docker image::
         
         docker pull firely/server:latest
+.. _vonk_releasenotes_5_5_0:
+
+Release 5.5.0, <put  date>, 2024
+---------------------------------
+
+Features
+^^^^^^^^
+#. Firely Server can now produce No-Op Outcome on resource update. 
```

Comment:
Quick question, what would be the effect of enabling the NoOp plugin for subscriptions? If there are no changes to the resource, would a subscriber still receive information on the update? It would be good to specify this in the subscription.rst file with a link to the information above.